examples/tutorials/self-paced-training/part-2_federated_learning_system/chapter-3_federated_computing_platform/03.1_federated_computing_architecture/system_architecture.ipynb

```diff
@@ -0,0 +1,152 @@
+    "\n",
+    "* **API Layer**: This is the API exposed to application developers, like Communicator and Cellnet.\n",
+    "* **Streamable Framed Message (SFM)**: This is the core of FCI and it provides abstraction on top of different communication protocols. It manages endpoints and connections.\n",
+    "* **Transport Drivers**: This layer is responsible for sending frames to other endpoints. It treats the frame as opaque bytes.\n",
+    "\n",
+    "<img src=\"./fci.png\" alt=\"FLARE Communication Interface\" width=\"300\" height=\"400\">\n",
+    "\n",
+    "## Federated Computing Architecture\n",
+    "\n",
+    "There are two parent control processes with corresponding job processes on each site. This enables support of concurrent, multi-job processes.\n",
+    "\n",
+    "<img src=\"./system_architecture.png\" alt=\"FLARE System Architecture\" width=\"700\" height=\"400\">\n",
+    "\n",
+    "## Federated Learning Framework\n",
+    "\n",
+    "Based on the basic core concepts, we have built many Federated learning workflows including FedAvg, FedOpt, FedProx, Scaffold, cyclic, swarming learning, split learning algorithms with many examples which can be found on the [website](https://nvidia.github.io/NVFlare/) and its [tutorial categories](https://nvidia.github.io/NVFlare/catalog/).\n",
+    "\n",
+    "## Enterprise Security and Privacy\n",
+    "\n",
+    "We have many features to support enterprise security as well as support privacy-enhancing technologies (PETs). Please refer to [Part-3 Security and Privacy](../../../part-3_security_and_privacy/part-3_introduction.ipynb).\n",
+    "\n",
+    "## Simulations\n",
+    "\n",
+    "We have built different tools for simulation including Python API and CLI. You have seen the Job API and simulator CLI in [Chapter-1](../../../part-1_federated_learning_introduction/Chapter-1_running_federated_learning_applications/01.0_introduction/introduction.ipynb).\n",
+    "\n",
+    "In [Section 3.2](../03.2_deployment_simulation/simulate_real_world_deployment.ipynb), we will also discuss how to simulate the deployment within a local machine.\n",
+    "\n",
+    "## Setup and Deployment\n",
+    "\n",
+    "Setting up the federated computing system is not a trivial task. We have built tools to make this process simpler. We will discuss this in [Chapter 4](../../chapter-4_setup_federated_system/04.0_introduction/introduction.ipynb).\n",
+    "\n",
+    "## Configuration\n",
+    "\n",
+    "NVFLARE supports several configuration formats: JSON, pyhocon, and YAML. You can see the details in [Configuration Files](https://nvflare.readthedocs.io/en/main/user_guide/configurations.html).\n",
+    "\n",
+    "You can also leverage the existing [job templates](https://github.com/NVIDIA/NVFlare/tree/main/job_templates): a set of predefined configurations and using [job CLI](https://github.com/NVIDIA/NVFlare/blob/main/examples/tutorials/job_cli.ipynb) to customize to your needs. \n"
```

Comment:
Shall we add here more details regarding templates, job CLI, jobAPI, and their relationships?